packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -287,60 +312,75 @@ async function doEmit(mainTspFile: string, emitter: Emitter) {
   /*Config emitter output dir and emit in tspconfig.yaml. */
   const defaultEmitOutputDirInConfig = `{output-dir}/{emitter-name}`;
   const tspConfigFile = path.join(baseDir, TspConfigFileName);
-  let configYaml = parseDocument(""); //generate a empty yaml
+  let configYaml = tryParseYaml("") ?? {}; //generate a empty yaml
```

Comment:
why not just ```configYaml = {}``` ?

Reply:
it will return an instance of Document which will contain some properties and function with implementation. We need a instance of Document.

Reply:
then why use {} as fallback?

Reply:
update to create Document instance directly